Game/Source/GDKTestGyms/Private/TestGymsCharacterMovementComp.cpp

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+
+#include "TestGymsCharacterMovementComp.h"
+#include "../GDKTestGymsCharacter.h"
```

Comment:
could we use a more absolute relative path here? E.g. one that does not rely on this file being in the directory above without a `..`.

Reply:
unfortunately the file exists in the root GDKTestGyms directory, which isn't included in any paths

Reply:
oh that isn't the projects root - can do actually